src/sql/src/parsers/copy_parser.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use snafu::ResultExt;
+use sqlparser::keywords::Keyword;
+
+use crate::error::{self, Result};
+use crate::parser::ParserContext;
+use crate::statements::copy::CopyTable;
+use crate::statements::statement::Statement;
+
+// COPY tbl TO 'output.parquet';
+impl<'a> ParserContext<'a> {
+    pub(crate) fn parse_copy(&mut self) -> Result<Statement> {
+        self.parser.next_token();
+        let copy_table = self.parse_copy_table()?;
+        Ok(Statement::Copy(copy_table))
+    }
+
+    fn parse_copy_table(&mut self) -> Result<CopyTable> {
+        let table_name =
+            self.parser
+                .parse_object_name()
+                .with_context(|_| error::UnexpectedSnafu {
+                    sql: self.sql,
+                    expected: "a table name",
+                    actual: self.peek_token_as_string(),
+                })?;
+
+        self.parser
+            .expect_keyword(Keyword::TO)
+            .context(error::SyntaxSnafu { sql: self.sql })?;
+
+        let file_name =
```

Comment:
I think we'd better follow pg's "COPY" grammar, that "COPY table_name TO file_name WITH FORMAT format_name". We can only deal with "format_name = parquet" now.